src/main/kotlin/org/jitsi/jicofo/bridge/ExternalBridgeSelectionStrategy.kt

```
@@ -0,0 +1,110 @@
+        requestBody["fallback_strategy"] = ExternalBridgeSelectionStrategyConfig.config.fallbackStrategy
+
+        val request = HttpRequest
+            .newBuilder()
+            .POST(HttpRequest.BodyPublishers.ofString(requestBody.toJSONString()))
+            .uri(URI.create(url))
+            .headers("Content-Type", "application/json")
+            .timeout(ExternalBridgeSelectionStrategyConfig.config.timeout)
+            .build()
+
+        val response: HttpResponse<String>
+        try {
+            response = httpClient.send(request, HttpResponse.BodyHandlers.ofString())
+        } catch (exc: Exception) {
+            logger.error("ExternalBridgeSelectionStrategy: HTTP request failed with ${exc}, using fallback strategy")
+            return fallback(bridges, conferenceBridges, participantRegion)
+        }
+
+        val statusCode = response.statusCode()
+        if (statusCode !in 200..299) {
+            logger.error("ExternalBridgeSelectionStrategy: HTTP request failed with ${statusCode}, using fallback strategy")
+            return fallback(bridges, conferenceBridges, participantRegion)
+        }
+
+        val responseBody: JSONObject
+        try {
+            responseBody = JSONValue.parseWithException(response.body()) as JSONObject
+        } catch (exc: Exception) {
+            logger.error("ExternalBridgeSelectionStrategy: HTTP response parsing failed with ${exc}, using fallback strategy")
+            return fallback(bridges, conferenceBridges, participantRegion)
+        }
+
+        val selectedBridgeIndex = responseBody["selected_bridge_index"] as? Number
+
+        if (selectedBridgeIndex == null) {
+            logger.error("ExternalBridgeSelectionStrategy: HTTP response selected_bridge_index missing or invalid, using fallback strategy")
+            return fallback(bridges, conferenceBridges, participantRegion)
+        }
+
+        val bridge = bridges!![selectedBridgeIndex.toInt()]
```

Can you fail early (before sending the request) if `bridges` is null or empty?